### PR TITLE
Screen: Confirmed New Infections (EXPOSUREAPP-4510)

### PR DESCRIPTION
**Pull request by jurajkusnier**
- Implementation of Infections Card

### CODE REVIEW

File: Corona-Warn-App/src/main/res/drawable/ic_statistics_info.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/res/drawable/ic_statistics_info.xml:1]: Icon is already there

File: Corona-Warn-App/src/main/res/layout/home_statistics_cards_infections_layout.xml [hunk 2]
```diff
@@ -11,15 +32,113 @@
...
+            android:id="@+id/tertiaryValue"
+            style="@style/StatisticsCardSecondaryValue"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@{total}"
+            tools:text="429.181" />
+
+        <ImageButton
+            android:id="@+id/statistics_card_info_button"
+            style="@style/Widget.AppCompat.ImageButton"
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/res/layout/home_statistics_cards_infections_layout.xml:132]: Use style StatisticsCardInfoButton

**Comment by chiljamgossow** [comment]
The primary label with the date is not correct. It should be Gestern/Yesterday

File: Corona-Warn-App/src/main/res/drawable/ic_statistics_info.xml (hunk 1 above)
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/res/drawable/ic_statistics_info.xml]: removed

**Comment by jurajkusnier** [comment]
> The primary label with the date is not correct. It should be Gestern/Yesterday

The data are clearly wrong, however, I implemented only card functionality and design in this PR. The card shows data it gets and since it worked with mock data, I guess the issue could be somewhere deeper.

File: Corona-Warn-App/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -402,12 +402,12 @@
         <item name="android:background">@drawable/contact_diary_card_ripple</item>
     </style>
 
-    <style name="StatisticsCardInfoButton" parent="buttonIcon">
-        <item name="android:height">@dimen/circle_icon_big</item>
-        <item name="android:width">@dimen/circle_icon_big</item>
+    <style name="StatisticsCardInfoButton" parent="@style/Widget.AppCompat.ImageButton">
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/res/values/styles.xml:405]: Please leave as it is. It's correct on statistics main branch

**Review by chiljamgossow — APPROVED** [review]
I think we should merge and fix issues separately if necessary

File: Corona-Warn-App/src/main/res/layout/home_statistics_cards_infections_layout.xml [hunk 4]
```diff
@@ -1,7 +1,28 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools">
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:ignore="MissingConstraints">
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/layout/home_statistics_cards_infections_layout.xml:5]: is this wanted?

File: Corona-Warn-App/src/main/res/layout/home_statistics_cards_infections_layout.xml [hunk 5]
```diff
@@ -11,15 +32,110 @@
         <TextView
             android:id="@+id/title"
             style="@style/headline5"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/spacing_normal"
+            android:layout_marginTop="@dimen/spacing_normal"
+            android:layout_marginEnd="0dp"
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/layout/home_statistics_cards_infections_layout.xml:39]: IMO this could be removed.

File: Corona-Warn-App/src/main/res/layout/home_statistics_cards_infections_layout.xml [hunk 6]
```diff
@@ -11,15 +32,110 @@
...
-            android:layout_marginStart="24dp"
-            android:layout_marginTop="24dp"
-            android:text="Infections"
+            android:layout_marginTop="@dimen/spacing_tiny"
+            android:importantForAccessibility="no"
+            android:src="@drawable/ic_main_illustration_infection"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="@id/layoutFlow" />
+
+        <androidx.constraintlayout.helper.widget.Flow
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/layout/home_statistics_cards_infections_layout.xml:55]: Why is this needed?

**Review by harambasicluka — APPROVED** [review]
LGTM, I just have 3 questions/remarks.

File: Corona-Warn-App/src/main/res/layout/home_statistics_cards_infections_layout.xml (hunk 4 above)
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/res/layout/home_statistics_cards_infections_layout.xml:5]: I use Flow widget to simplify the layout. Constraints are no longer necessary in this case, but lint check complains about it.

File: Corona-Warn-App/src/main/res/layout/home_statistics_cards_infections_layout.xml (hunk 6 above)
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/res/layout/home_statistics_cards_infections_layout.xml:55]: To simplify the layout. Flow widget is the way to go when we want to implement a kinda linear look in ConstraintLayout.

File: Corona-Warn-App/src/main/res/layout/home_statistics_cards_infections_layout.xml (hunk 5 above)
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/res/layout/home_statistics_cards_infections_layout.xml:39]: yes
